ads_async/bin/get.py

```diff
@@ -110,7 +110,7 @@ async def get_symbols(
 
 async def main(
     host: str,
-    symbols: List[str],
+    symbols: list[str],
     net_id: Optional[str] = None,
```

Comment:
Interesting that these optionals didn't get converted to `str | None` since `from __future__ import annotations` didn't exist in the file.  It's the correct thing to do, though inconsistent throughout the repo.

Reply:
That is interesting. Perhaps it's a pyupgrade bug.
I upgraded it and re-ran with the latest just as a check - and it picked up on different things 😁 aee0501

Reply:
It's still missing some of the `str | None` conversions, but after all our (my) fighting with `typing` I can see why they don't just assume that you can `from __future__ import annotations`.  👍